YarnSpinnerTests/TestBase.cs

```diff
@@ -39,9 +40,7 @@ public class TestBase
 
                 var testDataDirectory = path.Take(index).ToList();
 
-                var pathToTestData = string.Join(Path.DirectorySeparatorChar.ToString(), testDataDirectory.ToArray());
-
-                pathToTestData = Path.DirectorySeparatorChar + pathToTestData;
```

Comment:
The tests wouldn't run for me with this line. This may have broken something on not-Windows, but I'm currently in no position to test it.